parler_tts/modeling_parler_tts.py

```diff
@@ -65,6 +88,9 @@
 ]
 
 
+NEED_SETUP_CACHE_CLASSES_MAPPING = {"static": StaticCache, "sliding_window": SlidingWindowCache, "hybrid": HybridCache}
+QUANT_BACKEND_CLASSES_MAPPING = {"quanto": QuantoQuantizedCache, "HQQ": HQQQuantizedCache}
```

Comment:
Parler should only be compatible with static and sliding right? The latter only when we're doing ROPE in the attention
```suggestion
NEED_SETUP_CACHE_CLASSES_MAPPING = {"static": StaticCache, "sliding_window": SlidingWindowCache}
```